server/helpers/helpers_test.go

```diff
@@ -0,0 +1,46 @@
+package helpers
+
+import (
+	"os"
+	"testing"
+)
+
+func TestMergeStringMaps(t *testing.T) {
+	map1 := map[string]string{"key1": "value1", "key2": "value2"}
+	map2 := map[string]string{"key2": "override", "key3": "value3"}
+
+	expected := map[string]string{"key1": "value1", "key2": "override", "key3": "value3"}
+	result := MergeStringMaps(map1, map2)
+
+	if len(result) != len(expected) {
+		t.Errorf("Expected map length %d, got %d", len(expected), len(result))
+	}
+
+	for k, v := range expected {
+		if result[k] != v {
+			t.Errorf("Expected key %s to have value %s, but got %s", k, v, result[k])
+		}
+	}
+}
+
+func TestResolveFSRef(t *testing.T) {
```

Comment:
even if this increase code coverage I am not sure the test it self is relevant here and should have already been covered in meshkit

Reply:
+1 @lekaf974

Reply:
@ganeshvenkatasai PLease move to meshkit and also add file ref for "HTTP"